pkg/render/tiers/tiers.go

```diff
@@ -40,9 +43,13 @@ var DNSIngressNamespaceSelector = createDNSIngressNamespaceSelector(
 	render.IntrusionDetectionNamespace,
 	render.KibanaNamespace,
 	render.ManagerNamespace,
+	render.ECKOperatorNamespace,
+	render.PacketCaptureNamespace,
+	render.PolicyRecommendationNamespace,
 	common.TigeraPrometheusNamespace,
-	"tigera-skraper",
+	dpi.DeepPacketInspectionNamespace,
```

Comment:
Was this correctly allowed before somewhere else, or was this not being here a bug you're fixing along the way?

Reply:
I dont think we have deleted anything - just adding and also re-ordered. "tigera-skraper" moved to the end of the list.

Reply:
What I mean is - we've added dpi.DeepPacketInspectionNamespace to this policy.  That means one of a few things that I can think of:
1. The DPI namespace was previously not allowed to make DNS requests and that was fine because it never did (possibly even it wasn't used).
2. The DPI namespace was previously not allowed to make DNS requests by this policy, but there was some other policy somewhere else that allowed it.
3. The DPI namespace was previously not allowed to make DNS requests, and we never noticed that we'd broken it.

Basically, what I want to understand is:
- Why did you choose to add this line, and how did you discover that you needed to?
- What does that mean for DPI previously?  Was it broken?

Reply:
probably needs more investigation on dpi to be able to answer these questions.
based on the conversation we had, since dpi namespace is created by the operator, we include in the list to avoid any possible problems because of the denied access to dns / dns-cache.

Reply:
Sounds like we should omit dpi.DeepPacketInspectionNamespace for now since it previously didn't have any rules to allow DNS, so either:

- It didn't need DNS, and still doesn't.
- It does need DNS, but we weren't allowing it. This is a bug and we should fix in a separate PR.